ops/docker/rpc-proxy/nginx.template.conf

```diff
@@ -11,6 +11,10 @@ events {
 http {
   include    mime.types;
   index    index.html;
+  # The JSONRPC POST body must fit inside this allocation for the method parsing to succeed.
+  # https://github.com/openresty/lua-nginx-module#ngxreqread_body
+  # http://nginx.org/en/docs/http/ngx_http_core_module.html#client_body_buffer_size
+  client_body_buffer_size 128k;
```

Comment:
How can we make sure that we give better feedback for users if they send transactions that are too big? Although this likely makes the issue much less common, it may still be possible for someone to try to send a very large transaction and get an error back. It seems useful to give good feedback to the user so they don't complain on discord or whatever.

Reply:
Added a setting for client_max_body_size that will match the value of client_body_buffer_size.

 Requests above this limit will also be denied with an HTTP 413 response (entity too large)
 http://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size